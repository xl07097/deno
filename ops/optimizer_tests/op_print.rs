fn op_print(
  state: &mut OpState,
  msg: &str,
  is_err: bool,
) -> Result<(), AnyError> {
  // TODO(@littledivy): fast compatible https://github.com/denoland/deno/issues/17159
}
