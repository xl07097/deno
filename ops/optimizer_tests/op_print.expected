=== Optimizer Dump ===
returns_result: true
has_ref_opstate: true
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: true
fast_result: Some(Void)
fast_parameters: [V8Value, SeqOneByteString, Bool]
transforms: {1: Transform { kind: SeqOneByteString(Ref), index: 1 }}
is_async: false
fast_compatible: false
